docs/middleware/logger.md

```diff
@@ -88,6 +88,33 @@ app.Use(logger.New(logger.Config{
 app.Use(logger.New(logger.Config{
     DisableColors: true,
 }))
+
+// Use Custom Logger with Fiber Logger Interface
+type dumbLogger struct {
+	logger fiberlog.AllLogger
+    level  log.Level
+}
+
+func (l *dumbLogger) Write(p []byte) (n int, err error) {
+	switch l.level {
+	case log.LevelDebug:
+		l.logger.Debug(string(p))
+	case log.LevelError:
+		l.logger.Error(string(p))
+    }
+	return len(p), nil
+}
+
+func LoggerToWriter(customLogger fiberlog.AllLogger, level fiberlog.Level) io.Writer {
+	return &dumbLogger{
+        logger: customLogger,
+        level: level,
+    }
+}
```

Comment:
@haochunchang can we put that or a better generic wrapper in the code so that not every dev has to create this struct? or would that be too undynamic? 

what do you think? @gaby @efectn

Reply:
maybe we should recommend something like this ?

```go
func LoggerToWriter(customLogger fiberlog.AllLogger, level fiberlog.Level) io.Writer {
	return &struct {
		Write func(p []byte) (n int, err error)
	}{
		Write: func(p []byte) (n int, err error) {
			switch level {
			case fiberlog.LevelDebug:
				customLogger.Debug(string(p))
			case fiberlog.LevelError:
				customLogger.Error(string(p))
			}
			return len(p), nil
		},
	}
}
```
then we really don´t need extra code in the codebase

Reply:
I think we can provide the `LoggerToWriter` in the example to show how to use fiberlog interface in the middleware.

Reply:
ok @haochunchang pls adjust it a little bit

Reply:
I’ve tried to condensed into a single function wrapper that return io.Writer but it seems it still need to declare the `Write` method in order to satisfy the `io.Writer` interface.

Reply:
@haochunchang yeah, but in my example its there
so can you update the PR with this code snippet ?

Reply:
Do you mean adding it into the example?
Cause I failed to adopt this into the unit tests :sweat_smile:

Reply:
@haochunchang 
yeah add it in the readme in a new sub section